src/screens/Add.js

```diff
@@ -0,0 +1,192 @@
+                    }}>
+                    <BackIcon width={24} height={24} fill="#FFFFFF" />
+                  </ToolButton>
+                }
+              />
+              <ToolBar
+                button={
+                  <LanguageSwitcher
+                    firstLanguage={LANGUAGE_NAMES.EN}
+                    LeftText="MA"
+                    RightText="EN"
+                    color="#FFFFFF"
+                    iconType=""
+                    iconName="swap-horiz"
+                    onPress={() => null}
+                    iconSize={24}
+                  />
+                }
+              />
+              <ToolBar
+                button={
+                  <ToolButton onPress={action('clicked-add-button')}>
+                    <ModeIcon width={24} height={24} fill="#FFFFFF" />
+                  </ToolButton>
+                }
+              />
+            </View>
+            <View style={styles.heading}>
+              <SectionHeading text="Category: " />
+              <SelectDropdown
+                data={categories}
+                ref={dropdownRef}
+                defaultButtonText="Select Category"
+                buttonTextStyle={{
+                  color: isButtonEnable ? '#06B6D4' : '#111827',
+                  marginRight: -10,
+                  marginTop: -2,
+                  maxWidth: isButtonEnable ? '43%' : '100%',
+                  textAlign: 'left',
+                }}
+                dropdownIconPosition="right"
+                dropdownStyle={styles.dropDownPicker}
+                buttonStyle={[styles.pickerContainer]}
+                renderDropdownIcon={isButtonEnable ? WrappedIcon : null}
+                onSelect={selectedItem =>
+                  setSelectedCategoryValue(selectedItem.id)
+                }
+                buttonTextAfterSelection={selectedItem =>
+                  selectedItem.name[nativeLanguage]
+                }
+                rowTextForSelection={item => item.name[nativeLanguage]}
+              />
+            </View>
+            <View style={styles.englishField}>
+              <View style={styles.inputHeading}>
+                <SectionHeading text="The phrase in English: " />
+              </View>
+              <Textarea
+                editable={true}
+                phrase={englishPhrase}
+                onChange={text => setEnglishPhrase(text)}
+                placeholder="Enter here"
+              />
+            </View>
+            <View style={styles.malagasyField}>
+              <View style={styles.inputHeading}>
+                <SectionHeading text="The phrase in Malagasy: " />
+              </View>
+              <Textarea
+                editable={true}
+                phrase={malagasyPhrase}
+                onChange={text => setMalagasyPhrase(text)}
+                placeholder="Enter here"
+              />
+            </View>
+            <Button
+              isDisabled={isButtonEnable}
```

Comment:
this doesnt make sense, rename isButtonEnable to isButtonDisabled or refactor logic to be falsy
and do
`isDisabled={!isButtonEnable}`